feature/signupflow/signup/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <manifest package="com.epmedu.animeal.signup.signup" />
```

Comment:
can we leave signle `signup` in package?

Reply:
Done